app/app.go

```diff
@@ -9,276 +9,255 @@ import (
+	app.ProviderKeeper = provider.NewKeeper(cdc, keys[provider.StoreKey])
 
 	app.mm = module.NewManager(
-		genutil.NewAppModule(app.keeper.acct, app.keeper.staking, app.BaseApp.DeliverTx),
-		auth.NewAppModule(app.keeper.acct),
-		bank.NewAppModule(app.keeper.bank, app.keeper.acct),
-
-		supply.NewAppModule(app.keeper.supply, app.keeper.acct),
-		distr.NewAppModule(app.keeper.distr, app.keeper.acct, app.keeper.supply, app.keeper.staking),
-
-		mint.NewAppModule(app.keeper.mint),
-		slashing.NewAppModule(app.keeper.slashing, app.keeper.acct, app.keeper.staking),
-
-		staking.NewAppModule(app.keeper.staking, app.keeper.acct, app.keeper.supply),
+		genutil.NewAppModule(app.AccountKeeper, app.StakingKeeper, app.BaseApp.DeliverTx),
+		auth.NewAppModule(app.AccountKeeper),
+		bank.NewAppModule(app.BankKeeper, app.AccountKeeper),
+		crisis.NewAppModule(&app.CrisisKeeper),
+		supply.NewAppModule(app.SupplyKeeper, app.AccountKeeper),
+		gov.NewAppModule(app.GovKeeper, app.AccountKeeper, app.SupplyKeeper),
+		mint.NewAppModule(app.MintKeeper),
+		slashing.NewAppModule(app.SlashingKeeper, app.AccountKeeper, app.StakingKeeper),
+		distr.NewAppModule(app.DistrKeeper, app.AccountKeeper, app.SupplyKeeper, app.StakingKeeper),
+		staking.NewAppModule(app.StakingKeeper, app.AccountKeeper, app.SupplyKeeper),
+		evidence.NewAppModule(app.EvidenceKeeper),
 
 		// akash
-		deployment.NewAppModule(
-			app.keeper.deployment,
-			app.keeper.market,
-			app.keeper.bank,
-		),
-
-		market.NewAppModule(
-			app.keeper.market,
-			app.keeper.deployment,
-			app.keeper.provider,
-			app.keeper.bank,
-		),
-
-		provider.NewAppModule(app.keeper.provider, app.keeper.bank),
+		deployment.NewAppModule(app.DeploymentKeeper, app.MarketKeeper, app.BankKeeper),
+		market.NewAppModule(app.MarketKeeper, app.DeploymentKeeper, app.ProviderKeeper, app.BankKeeper),
+		provider.NewAppModule(app.ProviderKeeper, app.BankKeeper),
 	)
 
 	app.mm.SetOrderBeginBlockers(mint.ModuleName, distr.ModuleName, slashing.ModuleName)
 	app.mm.SetOrderEndBlockers(staking.ModuleName, deployment.ModuleName, market.ModuleName)
 
-	// NOTE: The genutils module must occur after staking so that pools are
-	//       properly initialized with tokens from genesis accounts.
+	// NOTE: The genutils moodule must occur after staking so that pools are
+	// properly initialized with tokens from genesis accounts.
 	app.mm.SetOrderInitGenesis(
-		distr.ModuleName,
-		staking.ModuleName,
-		auth.ModuleName,
-		bank.ModuleName,
-		slashing.ModuleName,
-		mint.ModuleName,
-		supply.ModuleName,
-		genutil.ModuleName,
-
-		// akash
-		deployment.ModuleName,
-		provider.ModuleName,
-		market.ModuleName,
+		auth.ModuleName, distr.ModuleName, staking.ModuleName, bank.ModuleName,
+		slashing.ModuleName, gov.ModuleName, mint.ModuleName, supply.ModuleName,
+		crisis.ModuleName, genutil.ModuleName, evidence.ModuleName,
+		deployment.ModuleName, provider.ModuleName, market.ModuleName, // akash
```

Comment:
why are you changing the format here?

Reply:
uses too many lines. I usually break the line at ~80 chars

Reply:
Please keep the previous format.